### ISSUE TITLE
Contributing?

### ISSUE THREAD

**Issue by michaelaird** [andry-tino/Rosetta#38]
My team and I have been hacking away at adding more conversions for C# features that we used in our Script# projects. We'd like to contribute it back into the main project. 

Please have a look at my branch and let me know what you think. There are several things that are still "work in progress" (like handling the case's inside a switch), but we've handled foreach loops, a rough handling of knockout observables and added a naive handling of csproj files.

There are a number of things that we've identified that we need and will try to implement

- Generics
- Convert List<x> to x[]
- Gracefully handle missing "this"
- Convert delegates to lambdas
- Allow for namespace instead of module
- Call "super" in inherited classes
- and more!

**Comment by andry-tino** [comment]
Hello @michaelaird. Sure thing, nice to have some help. Let me have a better look at your changes.

**Comment by michaelaird** [comment]
Great! We're new to using roslyn so feel free to make any "code review" type suggestions on things you think we should do differently.

**Comment by andry-tino** [comment]
Yes! In the meantime, please have a look at the Wiki I just updated in the section [Contribute guidelines](https://github.com/andry-tino/Rosetta/wiki/Contribute-guidelines). The section is still to be expanded, but it has already the basic info required for a good contribution.

I am reviewing the code in the meantime. Get back to you with feedback.
